openapi/api.yaml

```diff
@@ -81,6 +81,26 @@ paths:
           application/json:
             schema:
               $ref: '#/components/schemas/ComponentUsersRequest'
+  /spec_mentors:
+    get:
+      summary: List spec mentors and their activity
+      operationId: listSpecMentors
+      parameters:
+        - in: query
+          name: after
+          schema:
+            type: string
+            format: date
+      responses:
+        '200':
+          description: >-
+            List of all the matching spec mentors.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/SpecMentor'
```

Comment:
If you anticipate other consumers of this API, could you add a 400 response for the invalid `after` parameter case?